### DISCUSSION TITLE
Not sure how to access Budibase using the 1-click Digital Ocean Button

### DISCUSSION THREAD

**Discussion by christianechevarria** [Budibase/budibase#1633] [Help] [answered]
Hi everyone, I went ahead and used the one-click digital ocean button

![image](https://user-images.githubusercontent.com/23618650/120870257-4ffbfc80-c54d-11eb-9f01-dfe2901e4e54.png)
 
but I'm unable to access the server through the default budibase port, 10000, is there some additional set-up I have to do? 

After SSH-ing into the server I noticed that the default port for the application was also blocked by UFW so I went ahead and opened that up

After doing an `ls` on the home directory I didn't notice an environment vars file

![image](https://user-images.githubusercontent.com/23618650/120870446-cf89cb80-c54d-11eb-8eb4-b77b216ff072.png)

Is this what I'm missing?

**Comment by shogunpurple** [comment, marked as answer]
Hi @chrisgereina - the digitalocean one click deployment is currently outdated. We are currently still waiting on digitalocean to approve the latest image  after some issues on their side (should be the next few days). After that, the one click droplet will work out of the box.

To get your installation working now, you should be able to run the update.sh script in your droplet there. You may also need to replace the envoy.yaml and docker-compose.yaml in the droplet with the latest ones here:

https://github.com/Budibase/budibase/tree/master/hosting

**Reply by shogunpurple** [reply]
Failing that, you can follow the instructions in this other discussion here to get up and running:

https://github.com/Budibase/budibase/discussions/1569

**Reply by christianechevarria** [reply]
@shogunpurple thanks for the awesome answer, would the manual droplet setup instructions be up to date?

**Reply by christianechevarria** [reply]
As a quick update, the manual setup script for digital ocean leads to a 404   https://raw.githubusercontent.com/Budibase/budibase/master/hosting/bootstrap.sh

![image](https://user-images.githubusercontent.com/23618650/120875751-534db300-c562-11eb-9bba-9adab9bb5daf.png)

**Reply by shogunpurple** [reply]
Thanks @chrisgereina will update.

**Reply by shogunpurple** [reply]
@chrisgereina digitalocean image is updated a simple one click should work for you now!

**Reply by Atulfortune** [reply]
Hello, I am using the one-click deployment of digitalocean and I am facing the below issue. Kindly help me with it. 

![image](https://user-images.githubusercontent.com/31924556/125956993-e5ef2e60-3eb2-4e42-b64b-d4394dc33f20.png)

**Reply by shogunpurple** [reply]
@Atulfortune with the one click digitalocean droplet, your droplet should be good to go as soon as you deploy it. Just go to:

`<your-droplet-ip>:10000`

Does this not work?

**Reply by Atulfortune** [reply]
My bad I was watching and flowing this video "https://www.youtube.com/watch?v=2P4HaFkP6Zk"
<your-droplet-ip>:10000 is working.
Thank you for your quick response.

**Reply by kashoncode** [reply]
It seems like the Digital Ocean one click droplet isn't working again. Tried twice and both times I get connection refused on <droplet_ip>:10000.

Any ideas why this could be?

**Reply by danishfareed** [reply]
> It seems like the Digital Ocean one click droplet isn't working again. Tried twice and both times I get connection refused on <droplet_ip>:10000.
> 
> Any ideas why this could be?

Even same with me, I tried and it gives no response. there's not enough documentation for digital ocean.

**Reply by shogunpurple** [reply]
Hi folks - I just tried a one click droplet and it works as expected. 

- create new budibase droplet on DO 
- Wait 2-5 mins
- visit `<droplet-ip>:10000` in browser

It takes a few minutes to initialise, which may be what you are seeing if you try to access it straight away. Could you please provide some exact reproduction steps?

**Reply by EydlinIlya** [reply]
Hi. Looks like one-click app now working again
